content/kubernetes/re-clusters/auto-tiering.md

```diff
@@ -73,7 +73,7 @@ Below is an example REDB custom resource:
 apiVersion: app.redislabs.com/v1alpha1
 kind: RedisEnterpriseDatabase
 metadata:
-  name: rof-redb
+  name: autoteiring-redb
```

Comment:
@bascar can you confirm this?

Reply:
@AlonMagrafta @kaitlynmichael API is not changing, so this should not change.

Reply:
Actually, this is just a name, with no impact. Can stay this way, sorry.